lib/tether/tether_linux.go

```diff
@@ -68,9 +68,11 @@ func (t *tether) childReaper() error {
 
 	go func() {
 		var status syscall.WaitStatus
-
 		flag := syscall.WNOHANG | syscall.WUNTRACED | syscall.WCONTINUED
+
 		for range t.incoming {
+			log.Debugf("Woke up because of a received SIGCHLD")
```

Comment:
This message and "Inspecting children with status change" will always proc together - redundant